app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+import os
+
+
+class MvError(Exception):
+    pass
+
+
+def move_file(command: str) -> None:
+    _, source_file, path = command.split()
```

Comment:
Unpacking the command string should be done after validating its length. Move the unpacking line after the validation check to avoid potential errors if the command does not have exactly three elements.